test.js

```diff
@@ -125,7 +125,9 @@ test('clearQueue', t => {
 	const limit = pLimit(1);
 
 	Array.from({length: 1}, () => limit(() => delay(1000)));
-	Array.from({length: 3}, () => limit(() => delay(1000)));
+	Array.from({length: 3}, () => limit(() => delay(1000)).catch(
+		error => t.is(error.message, 'queue cleared before function was invoked')
+	));
```

Comment:
Looks like this new `t.is` assertion might not run if the promises don't reject, so maybe you should make the test stronger by ensuring they reject via [`t.throwsAsync`](https://github.com/avajs/ava/blob/master/docs/03-assertions.md#throwsasyncthrower-expectation-message) (or, perhaps, [via `t.plan` but I'm not sure](https://github.com/avajs/ava/blob/master/docs/recipes/when-to-use-plan.md))